pkg/etcd/etcd.go

```diff
@@ -53,42 +87,38 @@ type learnerProgress struct {
 	LastProgress     metav1.Time `json:"lastProgress,omitempty"`
 }
 
+// Members contains a slice that holds all
+// members of the cluster.
+type Members struct {
+	Members []*etcdserverpb.Member `json:"members"`
+}
+
+// newClient create a new Kubernetes client from configuration.
+func newClient(kubeConfigPath string, k8sWrapTransport transport.WrapperFunc) (*kubernetes.Clientset, error) {
```

Comment:
Since you're not using the second parameter, you could probably just move this into a util package and consolidate: https://github.com/k3s-io/k3s/blob/8f1a20c0d32c957c8ef953ad44bac8f9f53220e0/pkg/agent/run.go#L136